src/receivers/AwsLambdaReceiver.ts

```diff
@@ -10,15 +10,15 @@ import { StringIndexed } from '../types/helpers';
 
 export interface AwsEvent {
   body: string | null;
-  headers: any;
+  headers: Record<string, string>;
```

Comment:
Breaking change: HTTP request headers are string:string maps

Reply:
I've created a draft PR _into_ this PR to be able to more specifically discuss this one change here: https://github.com/slackapi/bolt-js/pull/2277

Reply:
Thanks for quickly resolving this! Yes, I agree that this type of change is fine when we release a new major version.